### ISSUE TITLE
Wrong ICS appear to be in relaton-jis records

### ISSUE THREAD

**Issue by opoudjis** [relaton/relaton-jis#4]
E.g. if I retrieve JIS Z 8002

````
[relaton] ERROR: JIS Z 8002 -- No such file or directory @ rb_sysopen - /Users/nickn/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/gems/isoics-0.1.11/lib/isoics/../../ics/01_040_01
                                                                                    01.json
````

Appears to be happening with most JIS records.

This appears to be forming the filename as "01_040_01\n01.json" (!)

**Comment by ronaldtse** [comment]
This looks like a bug in the retrieval of ICS codes in Relaton-JIS. We should also update the isoics gem so it is more resistent to input error.

**Comment by andrew2net** [comment]
fixed in v 1.14.3
